plugins/BEdita/API/src/Auth/EndpointAuthorize.php

```diff
@@ -123,12 +135,13 @@ public function authorize($user, ServerRequest $request)
      * Perform user unauthentication to return 401 Unauthorized
      * instead of 403 Forbidden
      *
-     * @return void
+     * @return mixed
      */
     protected function unauthenticate()
     {
         $controller = $this->_registry->getController();
-        $controller
+
+        return $controller
```

Comment:
That's right, I did it just to make sure code coverage is 100%, maybe there's a better way to do it

Removing it anyway